tensorflow/g3doc/api_docs/python/nn.md

```diff
@@ -1209,8 +1209,7 @@ Performs the max pooling on the input.
 Performs max pooling on the input and outputs both max values and indices.
 
 The indices in `argmax` are flattened, so that a maximum value at position
-`[b, y, x, c]` becomes flattened index
-`((b * height + y) * width + x) * channels + c`.
+`[b, y, x, c]` becomes flattened index `(y * width + x) * channels + c`.
```

Comment:
The correct place to update the docstring would be here: 
￼https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/nn_ops.cc#L1305